packages/labs/examples/selectExample.tsx

```diff
@@ -62,7 +62,7 @@ export class SelectExample extends BaseExample<ISelectExampleState> {
                 itemRenderer={this.renderFilm}
                 noResults={<MenuItem disabled={true} text="No results." />}
                 onItemSelect={this.handleValueChange}
-                popoverProps={{ popoverClassName: minimal ? Classes.MINIMAL : "" }}
+                popoverProps={{ minimal }}
```

Comment:
check it out: this is the fix for that pesky 4px gap! as in, we already had a fix in this new `minimal` prop (instead of the className-only approach on the left), just had to use it properly. 

cc @brieb

Reply:
Oh nice -- thanks! :)